guides/common/assembly_monitoring-hosts-using-red-hat-insights.adoc

```diff
@@ -4,6 +4,16 @@ ifdef::orcharhino[]
 include::modules/proc_installing-red-hat-cloud-plug-in.adoc[leveloffset=+1]
 endif::[]
 
-include::modules/proc_using-red-hat-insights.adoc[leveloffset=+1]
+include::modules/con_monitoring_hosts_using-red-hat-insights.adoc[leveloffset=+1]
```

Comment:
I meant moving the content into the [existing `con_` module](#diff-098942e9a92703e3dcbfa1053dbf28ed3f00c5a9aa1873c034b19604c886ea6eR1) that is already included at the beginning of this assembly instead of adding a new one of almost the same name. Please, redo.